src/types_utils/camel-case.ts

```diff
@@ -0,0 +1,47 @@
+/** Utility types to enable conversion of type properties to camelCase
```

Comment:
Does this file still need to be in here?

Reply:
I think we can keep this file until we change its behavior so that it won't hurt `SlackManifestType`  discriminated union or decide how we going to handle the case conversions (snake to camel case).